12LessonTwelve/homework/css/style.css

```diff
@@ -0,0 +1,159 @@
+body {
+  background: #000;
+}
+img {
+  display: block;
+  max-width: 100%;
+  height: auto;
+}
+.container {
+  background: #fff;
+}
+header {
+  font-family: 'Lily Script One', cursive;
+  height: 200px;
+  text-align: center;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-size: 2em;
+}
+section {
+  overflow: auto;
+}
+article {
+  margin-bottom: 60px;
+}
+article > div {
+  font-family: sans-serif;
+  text-align: center;
+  border: 1px solid #a5a5aa;
+  margin: 0 3%;
+}
+.img {
+  position: relative;
+  margin: 0;
+}
+.amount {
+  background: #66D9C1;
+  border:1px solid #5dc1ab;
+  color: #fff;
+  font-weight: 700;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 0;
+  position: absolute;
+  right: 0;
+  bottom: 0;
+  height: 30px;
+  width: 100px;
+}
+article h3 {
+  font-size: 0.80em;
+  font-weight: 600;
+  padding: 15px 0;
+  border-bottom: 1px solid #a5a5aa;
+}
+article a {
+  display: inline-block;
+  color: #fff;
+  text-decoration: none;
+  text-transform: uppercase;
+  height: 40px;
+  line-height: 40px;
+  width: 110px;
+  margin: 30px 0;
+  background: #343436;
+  font-size: 0.70em;
+  font-weight: 600;
+  border: 1px solid #a5a5aa;
+}
+.sale {
+  position: absolute;
+  top: 0;
+  left: 0;
+  margin: 0;
+  height: 30px;
+  line-height: 30px;
+  width: 50px;
+  text-transform: uppercase;
+  color: white;
+  background: #F04933;
+  border: 1px solid #dd4433;
+  font-size: 0.80em;
+  font-weight: 700;
+  text-align: center;
+}
+.price-change {
+  display: flex;
+  justify-content: space-around;
+  width: 110px;
+}
+.price-change span {
+  font-size: 0.95em;
+  text-decoration: line-through;
+}
+@media screen and (min-width: 320px) {
+  .container::before {
+    content: "320px";
+    color: green;
+    position: absolute;
+    top: 0;
+    left: 0;
+    font-weight: 700;
+    margin: 5px;
+    font-size: 2em;
+  }
+}
+@media screen and (min-width: 480px) {
+  .container::before {
+    content: "480px";
+    color: maroon;
+  }
+  article {
+    margin-left: 9%;
+    margin-right: 9%;
+  }
+}
+@media screen and (min-width: 620px) {
+  .container::before {
+    content: "620px";
+    color: orange;
+  }
+  section {
+    display: flex;
+    flex-flow: wrap;
+  }
+  article {
+    margin: 0 auto;
+    margin-bottom: 60px;
+    flex-basis: 300px;
+  }
+  .amount {
+    width: 70px;
+  }
+}
+@media screen and (min-width: 810px) {
+  .container::before {
+    content: "810px";
+    color: black;
+  }
+  article {
+    flex-basis: 350px;
```

Comment:
Try using a percentage instead. Since you want 4 objects side by side, ~25% (pending margin/padding) should do it.